src/m_obj.c

```diff
@@ -892,6 +893,29 @@ int obj_nsigoutlets(const t_object *x)
     return (n);
 }
 
+// DRAW MULTICHANNEL CONNECTIOSN
+void obj_sigoutletsetchans(const t_object *x, int m, int nchans){
+    int n;
+    t_outlet *o2;
+    for (o2 = x->ob_outlet, n = 0; o2 && m--; o2 = o2->o_next);
+    if (o2 && (o2->o_sym == &s_signal))
+    {
+        o2->o_nchans = nchans;
+    }
+}
+
+int obj_sigoutletgetchans(const t_object *x, int m){
+    int n;
+    t_outlet *o2;
+    for (o2 = x->ob_outlet, n = 0; o2 && m--; o2 = o2->o_next);
+    if (o2 && (o2->o_sym == &s_signal))
+    {
+        return o2->o_nchans;
+    }
```

Comment:
Same as above. Also, what's the return value if the if-statement is false? ;-) Actually, the compiler should give you a warning about missing return value!

I would do the following:
```
    if (o2 && (o2->o_sym == &s_signal))
        return o2->o_nchans;
    else
        return 0;
```
See also https://github.com/pure-data/pure-data/pull/2057/files#diff-5dcf32c92a1ab6eae741228954925258717e876e91c08deff0ce611f94833d14L2400